explorer_frontend/src/features/account/components/AccountInfo.test.tsx

```diff
@@ -1,62 +1,82 @@
 import { renderWithLayout } from "@test/unit/renderWithLayout";
 import { screen } from "@testing-library/react";
 import * as effectorReact from "effector-react";
-import { describe, it, vi } from "vitest";
+import { describe, it } from "vitest";
 import { measure } from "../../shared/utils/measure";
 import { AccountInfo } from "./AccountInfo";
-
-// biome-ignore lint/suspicious/noExplicitAny: <explanation>
-const getUseUnitReturnedValue = (args: any = {}): any => {
-  return [
-    args.account ?? { address: "0x123", balance: "1000" },
-    args.accountCometaInfo ?? null,
-    args.isLoading ?? false,
-    args.isLoadingCometaInfo ?? false,
-    args.params ?? { address: "0x123" },
-    args.cometa ?? null,
-  ];
+import { fork, type Scope, type TypeOfSource } from "effector";
+import {
+  $account,
+  $accountCometaInfo,
+  loadAccountCometaInfoFx,
+  loadAccountStateFx,
+} from "../model";
+import { $cometaClient } from "../../cometa/model";
+import "../init.ts";
+import { router, addressRoute } from "../../routing";
+import { RouterProvider } from "atomic-router-react";
+
+const initScope = (args?: {
+  account?: Partial<TypeOfSource<typeof $account>>;
+  accountCometaInfo?: Partial<TypeOfSource<typeof $accountCometaInfo>>;
+  isLoading?: boolean;
+  isLoadingCometaInfo?: boolean;
+  params?: Partial<TypeOfSource<typeof addressRoute.$params>>;
+  cometa?: Partial<TypeOfSource<typeof $cometaClient>>;
+}): Scope => {
+  return fork({
+    values: [
+      [$account, args?.account ?? { address: "0x123", balance: "1000" }],
+      [$accountCometaInfo, args?.accountCometaInfo ?? null],
+      [loadAccountStateFx.pending, args?.isLoading ?? false],
+      [loadAccountCometaInfoFx.pending, args?.isLoadingCometaInfo ?? false],
+      [addressRoute.$params, args?.params ?? { address: "0x123" }],
+      [addressRoute.$isOpened, true],
+      [$cometaClient, args?.cometa ?? null],
+    ],
+  });
 };
 
-vi.mock("../../routing", () => ({
-  addressRoute: {
-    $params: { getState: () => ({ address: "0x123" }) },
-    $isOpened: { getState: () => true },
-  },
-}));
-
-vi.mock("effector-react");
-const mockUseUnit = vi.mocked(effectorReact.useUnit);
-
 describe("AccountInfo", () => {
-  afterEach(() => {
-    vi.clearAllMocks();
-  });
-
   it("renders without crashing", () => {
-    mockUseUnit.mockReturnValue(getUseUnitReturnedValue());
-
-    renderWithLayout(<AccountInfo />);
+    const scope = initScope();
+    renderWithLayout(
+      <effectorReact.Provider value={scope}>
+        <RouterProvider router={router}>
```

Comment:
What if we move <RouterProvider> to default layout?

Reply:
Great idea